### ISSUE TITLE
Motif presence at genomic location

### ISSUE THREAD

**Issue by dborgesr** [davek44/Basset#28]
In my use case of Basset i've bumped into something that would be super useful. This may already be doable (but i'm not sure how): a method to take in a genomic region, a target cell type and output what motifs are associated with that region in that cell type using Tomtom. I know that right now you can feed the entire model and sequences but i'm not sure how to hook all that together.

**Comment by davek44** [comment]
Yea this has been on my radar for awhile. I suppose I've hesitated because tomtom is so often unable to identify what proteins are most relevant to the motif. I'll post here if I code up a solution.

**Comment by dborgesr** [comment]
Yeah, but I can definitely say that this is something that is absolutely crucial for this to catch real traction, need to connect to previous people's stuff. I started working on this and it should just be feeding the model the seqeunce of a specific location right?

**Comment by davek44** [comment]
Just to make sure that we're on the same page- you want to annotate the motifs that pop up with loss scores in the saturated mutagenesis heat maps, right? For example, the sequence in Figure 5 in the paper would be annotated as CTCF. That's how I've been thinking about it.

Obviously, there are already tools for annotating sequences with motifs, like Tomtom. Basset can help suggest which motifs are actually relevant. So maybe the easiest approach would be to add an option to basset_sat.py to query the sequence for significant motifs with Tomtom, but filter the list for only those that overlap a nucleotide with a loss score above some threshold.
